### PR TITLE
Show Project details in a pop-up

### PR DESCRIPTION
**Pull request by jonas-45**
The Pop-up-branch has added the following features to the project:

-Contents of the project section are stored in an array and populated dynamically.
-A click on the see project button brings up a pop-up window to display the project details in both mobile and desktop modes.
-When the close icon on the pop-up is clicked, it closes the pop-up window

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1141,32 +1108,188 @@ nav h2 {
...
+    display: flex;
+    width: 100%;
+    max-height: 556px;
+  }
+
+  .work-description-container {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 10px;
+  }
```
Comment (GedeonTS) [inline style.css:1218]: - [ ] Kindly make sure your design matches the Figma design, by aligning the text below the image and displaying tech stacks and the action button to the bottom left. :recycle: 
- Your design:
yourDeskP
 - Figma design:
FigmaDESK

File: style.css [hunk 2]
```diff
@@ -1141,32 +1108,188 @@ nav h2 {
...
+.popup-statusbar {
+  display: block;
 }
 
-.button {
-  width: 60px;
-  height: 60px;
-  background: #a11692;
+.popup-button-line {
+  display: block;
```
Comment (GedeonTS) [inline style.css:1142]: - [ ] :recycle: Please make sure the buttons are centered and the text "Live demo" is on one line as per the Figma design below 
- your design:
yourMobLinks

- Figma:
figbtns

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: :

Hi Team, this is @GedeonTS your reviewer, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights :tophat:  :+1: 
- No linter errors :+1: 
- Good use of Github flow :+1: 
- The details popup is working :+1: 
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [ OPTIONAL ] Please make sure your current work doesn't alter your previous :pray:  ,  I noticed that your popup menu is no longer working on this branch, kindly fix it to make the project complete :100: 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @jonas-45, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Highlights:
- All linters passing
- All requirements completed
- Professional readme

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**